go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/sigstore/cosign/v2
 
-go 1.23.2
-
-toolchain go1.23.4
+go 1.23.4
```

Comment:
Should we drop this down to 1.23.0 and do the same thing we did in sigstore/sigstore to test against the latest Go?

Reply:
i tried but when running go mod tidy it update to 1.23.4 i think some dependencies are set to 1.23.4